packages/at_auth/lib/at_auth.dart

```diff
@@ -1,18 +1,26 @@
 library at_auth;
 
+import 'package:at_auth/src/auth_interface.dart';
+import 'package:at_auth/src/auth_interface_impl.dart';
+
 export 'src/at_auth_base.dart';
-export 'src/at_auth_impl.dart';
-export 'src/auth_constants.dart';
-export 'src/onboard/at_onboarding_request.dart';
-export 'src/onboard/at_onboarding_response.dart';
 export 'src/auth/at_auth_request.dart';
 export 'src/auth/at_auth_response.dart';
-export 'src/keys/at_auth_keys.dart';
-export 'src/exception/at_auth_exceptions.dart';
+export 'src/auth_constants.dart';
 export 'src/enroll/at_enrollment_base.dart';
-export 'src/enroll/at_enrollment_impl.dart';
-export 'src/enroll/at_enrollment_response.dart';
+export 'src/enroll/at_enrollment_notification_request.dart';
 export 'src/enroll/at_enrollment_request.dart';
+export 'src/enroll/at_enrollment_response.dart';
 export 'src/enroll/at_initial_enrollment_request.dart';
 export 'src/enroll/at_new_enrollment_request.dart';
-export 'src/enroll/at_enrollment_notification_request.dart';
+export 'src/enroll/base_enrollment_request.dart';
+export 'src/enroll/enrollment_request.dart';
+export 'src/enroll/enrollment_request_decision.dart';
+export 'src/enroll/enrollment_server_response.dart';
+export 'src/enroll/first_enrollment_request.dart';
+export 'src/exception/at_auth_exceptions.dart';
+export 'src/keys/at_auth_keys.dart';
+export 'src/onboard/at_onboarding_request.dart';
+export 'src/onboard/at_onboarding_response.dart';
+
+final AtAuthInterface atAuthBase = AtAuthInterfaceImpl();
```

Comment:
Why is this final? Why is this variable being created at all?

Reply:
I *think* this is helpful to the user of the package. However please add some comments indicating that the main place to start is 'AtAuthInterface'